modules/dmrpp_module/build_dmrpp_util.cc

```diff
@@ -28,6 +28,8 @@
 #include <memory>
 #include <iterator>
 #include <unordered_set>
+#include <iomanip>      // std::put_time()
+#include <time.h>      // std::gmtime_r()
```

Comment:
I think the time.h include should be 'ctime.' Using ctime guarantees the names are in the std namespace while with time.h, it's only 'likely' they are.